src/howto/assembly_levels.md

```diff
@@ -0,0 +1,55 @@
+
+The different assembly levels are accessed through the following unified interface:
+```c++
+AssemblyLevel assembly_level = ...;
+a->SetAssemblyLevel(assembly_level);
+```
+where `a` is either an `mfem::BilinearForm`, `mfem::MixedBilinearForm`,
+`mfem::DiscreteLinearOperator`, or `mfem::NonlinearForm`.
+
+## Assembly levels and backends
+
+MFEM integrates two backends that interact with the assembly levels, namely the OCCA backend and the libCEED backend.
+Backends are accessible by configuring the `mfem::Device` accordingly.
+The native MFEM backend, OCCA backend, and libCEED backend each offer different features, and support different Integrators.
+
+| Supported Integrators             | native MFEM | OCCA backend | libCEED backend |
+|-----------------------------------|-------------|--------------|-----------------|
+| Mass Integrator                   | V           | V            | V               |
+| Vector Mass Integrator            | V           | X            | V               |
+| Vector FE Mass Integrator         | V           | X            | X               |
+| Convection Integrator             | V           | X            | V               |
+| Non-linear Convection Integrator  | V           | X            | V               |
+| Diffusion Integrator              | V           | V            | V               |
+| Vector Diffusion Integrator       | V           | X            | V               |
+| DGTrace Integrator                | V           | X            | X               |
+| Mixed Vector Gradient Integrator  | V           | X            | X               |
+| Mixed Vector Curl Integrator      | V           | X            | X               |
+| Mixed Vector Weak Curl Integrator | V           | X            | X               |
+| Gradient Integrator               | V           | X            | X               |
+| Vector Divergence Integrator      | V           | X            | X               |
+| Vector FE Divergence Integrator   | V           | X            | X               |
+| Curl Curl Integrator              | V           | X            | X               |
+| Div Div Integrator                | V           | X            | X               |
+
+| Features                          | native MFEM | OCCA backend | libCEED backend |
+|-----------------------------------|-------------|--------------|-----------------|
+| Tensor elements support           | V           | V            | V               |
+| Simplices support                 | X           | X            | V               |
```

Comment:
```suggestion
| Simplices support                 | ❌            | ❌             | ✅                |
```

Reply:
Excellent work.  I like the detail and the overview table.